hosts

```diff
@@ -194,108 +162,147 @@ ioc_ps_srv
 #   li_ps_diags       : LI power supply diagnostics IOCs
 #   bo_ps_diags       : BO power supply diagnostics IOCs
 
+# fac-testes-CA.abtlus.org.br (VM)
+# ================================
+# 	si_ps_trims_m12   : SI trims from sectors M1 and M2
+# 	as_ap_sofb        : TB, TS and BO SOFB IOCs
+#   as_ap_diag        : Diagnostics
+#                       - sirius-ioc-as-pu-diag.service
+#                       - sirius-ioc-as-rf-diag.service
+#                       - sirius-ioc-li-ap-diag.service
+#   si_ap_bl          : Beamline image IOC
+#                       - sirius-ioc-si-ap-manaca.service
+#   as_ti             : high level timing for all accelerators
+#   as_ap             : - sirius-ioc-li-ap-energy.service
+#                       - sirius-ioc-li-ap-currinfo.service
+#                       - sirius-ioc-bo-ap-currinfo.service
+#                       - sirius-ioc-ts-ap-currinfo.service
+#                       - sirius-ioc-si-ap-currinfo.service
+#                       - sirius-ioc-si-ap-currinfo-lifetime.service
+#                       - sirius-ioc-tb-ap-posang.service
+#                       - sirius-ioc-ts-ap-posang.service
+#                       - sirius-ioc-bo-ap-tunecorr.service
+#                       - sirius-ioc-bo-ap-chromcorr.service
+#                       - sirius-ioc-si-ap-tunecorr.service
+#                       - sirius-ioc-si-ap-chromcorr.service
+#                       - sirius-ioc-as-ap-machshift.service
+
+# lnls449-linux
+# =============
+
+# lnls454-linux
+# =============
+
+# lnls451-linux
+# =============
+
+# lnls452-linux
+# =============
+
+# lnls455-linux
+# =============
 
 # --- AS
 
 [ioc_as_ti_srv]
-lnls451-linux
+fac-testes-CA.abtlus.org.br
 
 [ioc_as_ap_srv]
-lnls451-linux
+fac-testes-CA.abtlus.org.br
 
 [ioc_as_ps_dclinks_srv]
-lnls451-linux
+lnlsfac-srv2
 
 [ioc_as_ap_sofb_srv]
-lnls449-linux
+fac-testes-CA.abtlus.org.br
 
 [ioc_si_ap_bl_srv]
-lnls455-linux
+fac-testes-CA.abtlus.org.br
 
 [ioc_as_ap_diag_srv]
-lnls455-linux
+lnlsfac-srv2
```

Comment:
considering the proposed config above, `ioc_as_ap_diag_srv` would be at `fac-testes-CA.abtlus.org.br`, doesn't it?

Reply:
@anacso17 , fixed it but also changes the IOC distribution. please, take a look.